treeopt/modules/sampling.py

```diff
@@ -0,0 +1,37 @@
+import smt.sampling_methods as smtSam
+
+def latinHypercube(limits, ndoe):
+    """Function to approach a designspace by using a latin hypercube algorithm
+    INPUT:
+        limits: numpy array representing the limit of the desingnspace
+        ndoe: number of samplingpoints to be generated in the designspace
+    OUTPUT:
+        Numpy-array containg the sampled points
+    """
+    sampling = smtSam.LHS(xlimits = limits, criterion='m')
+    x = sampling(ndoe)
+    return(x)
+    
+def fullFaktorial(limits, ndoe):
```

Comment:
factorial with a c instead of a k. Also please use an underscore and lower case letters.